libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2493,7 +2497,8 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
     ],
     keywords: ["xlm", "stellar"],
   },
-  osmosis: {
+  // osmosi
+  osmo: {
```

Comment:
Are we sure that we want to rename osmosis here ? Are we sure we prevent all side effect on renaming this ?

Reply:
AFAIK no risk as the keys are only use to access the objects themselves from the ID

Reply:
Could solve the issue differently so I reverted this part